src/viking.js

```diff
@@ -1,11 +1,79 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    
+    constructor(health, strength){
+        this.health=health;
+        this.strength=strength;
+        }
+    attack(){
+        return this.strength;
+            }
+    receiveDamage(damage){
+    this.health-damage;
+           }
 
-// Viking
-class Viking {}
+}
 
+// Viking
+class Viking extends Soldier {
+    constructor(name, health, strength){
+        super(health,strength);
+        this.name=name;
+        
+                                       }
+    receiveDamage(damage){
+      
+        let status=this.health-damage;
+        if(status>0){
+            return `"${this.name} has received ${damage} points of damage"`;
```

Comment:
You have extra quotations here

```suggestion
            return `${this.name} has received ${damage} points of damage`;
```